submissions/YuliiaHordiichuk/js-dom/script.js

```diff
@@ -0,0 +1,122 @@
+        img: './img/american.jpg',
+        subtitle: 'Breed Overview',
+        details: ['Height: 8 to 10 inches', 'Weigth: 10 to 15 pounds', 'Physical Characteristics: athletic build; short, dense coat']
+    },
+    {
+        title: 'Balinese',
+        description: 'Balinese are known for their intelligence, their friendly, inquisitive and playful nature, and their striking beauty. They are also often vocal and rather demanding. The breed arose as a result of a spontaneous genetic mutation in pure breed Siamese that caused the cats to develop a longer coat type.',
+        img: './img/balinese.jpg',
+        subtitle: 'Breed Overview',
+        details: ['Height: 10 to 12 inches', 'Weigth: 6 to 11 pounds', 'Physical Characteristics: athletic build; short, heavy coat']
+    },
+    {
+        title: 'British Shorthair',
+        description: `The British shorthair is an extremely adaptable cat that can thrive in an apartment. These quiet, laid-back cats typically prefer lounging over racing around their homes. But they can have bursts of energy and love interactive play to challenge them mentally and physically. Food puzzles are great to keep them entertained in a small space, as long as they don't lead to overfeeding.`,
+        img: './img/british.jpg',
+        subtitle: 'Breed Overview',
+        details: ['Height: 12 to 14 inches', 'Weigth: 7 to 17 pounds', 'Physical Characteristics: large head; round eyes; well-balanced body']
+    }
+]; 
+
+const navElement = document.querySelector('.nav'); 
+
+const createButton = (name) => {
+    return `
+    <li class ="nav__item">
+        <button class="nav__btn">${name}</button>
+    </li>
+    `;
+}; 
+
+const getButtons = () => {
+    return data.map(obj => createButton(obj.title)); 
+}; 
+
+
+const insertNav = () => {
+    const ul = document.querySelector('.nav__list'); 
+    ul.innerHTML = getButtons().join('');
+};
+
+
+const insertData = (title) => {
+    const main = document.querySelector('main'); 
+    const { img, description, subtitle, details } = data.filter(catObj => catObj.title === title)[0];
+
+    const characteristicsAsString = details.map(detail => {
+        return  `<li class='overview__item'>${detail}</li>`
+    }).join('');
+
+    main.innerHTML = `
+        <div class="content">
+            <img class="content__img" src=${img} alt=${title}>
+            <div class="content__text">
+                  <h2 class = "content__title">${title}</h2>
+                  <p class = "content__desc">${description}</p>
+            </div>   
+        </div>
+        <div class= "overview">
+            <h3 class="overview__title">${subtitle}</h3>
+            <ul class= "overview__list">${characteristicsAsString}
+            </ul>
+        </div>
+    `; 
+}
+
+const removeActiveClasses = () => {
```

Comment:
Do you really need to loop through the all of your navigation elements.
As I see you have only one active element.
You can do it using only one operation:
`document.querySelector('.nav__btn.active').classList.remove('active');`